benchmarks/run_bevt_ootb_train.sh

```diff
@@ -0,0 +1,36 @@
+NODES=1
+NUM_GPUS=8
+BS=8
+EP=8
```

Comment:
We were previously setting EP=1. 
cc: @zstreet87

Reply:
Scripts directly under `benchmarks/` are not finalized, please refer to `benchmarks/bevt/ootb/run_bevt_train.sh`.